### ISSUE TITLE
Apparent error in sample JSON file for Other License fields

### ISSUE THREAD

**Issue by swinslow** [spdx/spdx-spec#518]
Looking at the sample JSON file's contents for the Beer-Ware license, I see:

https://github.com/spdx/spdx-spec/blob/3dddfc34d7e770d4c17fc19f61be8554e3934c3d/examples/SPDXJSONExample-v2.2.spdx.json#L21-L24

I know this is just sample Extracted Text and so it doesn't really matter what it says. However, looking at what's there in the string, the Extracted Text appears to contain `LicenseName:`, `LicenseCrossReference:`, and `LicenseComment:` tags and values within the text string.

I'm guessing those were perhaps incorrectly interpreted by the converter that created this JSON file, as being part of the Extracted Text instead of being separate tags themselves. As a result, this license also doesn't actually contain those fields.

Not sure if this is an issue with the converter tool, or just with this example file -- so I'm not posting a PR to fix it, but did want to flag the issue.

**Comment by goneall** [comment]
@swinslow Good catch!  

In looking that the tag/value file I'm concluding this was an error in manually updating the example file.  Notice that the `</text>` is after the other fields.  There is also a very suspicious `</` at the end of the actual text.

Once we merge in PR #509 I can fix this - I don't want to fix it now and cause merge conflicts.

```
LicenseID: LicenseRef-Beerware-4.2
ExtractedText: <text>"THE BEER-WARE LICENSE" (Revision 42):
phk@FreeBSD.ORG wrote this file. As long as you retain this notice you
can do whatever you want with this stuff. If we meet some day, and you think this stuff is worth it, you can buy me a beer in return Poul-Henning Kamp  </
LicenseName: Beer-Ware License (Version 42)
LicenseCrossReference:  http://people.freebsd.org/~phk/
LicenseComment: 
The beerware license has a couple of other standard variants.</text>
```

**Comment by goneall** [comment]
@swinslow Let me know if you're OK with me merging #509

**Comment by swinslow** [comment]
Thanks @goneall! I just went and gave a +1 for #509.

Yup -- I stumbled across this one while doing some testing with converting the sample JSON file to tag-value. It outputted the full Extracted Text in the form you pasted above -- and I couldn't for the life of me figure out how the `<text>` and `</text>` flags were doing the things they seemed to be doing...  :)

**Comment by goneall** [comment]
@swinslow I just submitted PR #519 which should resolve this

**Comment by goneall** [comment]
Resolved with #519
